### DISCUSSION TITLE
On-screen display problem

### DISCUSSION THREAD

**Discussion by Nick222** [widelands/wl_addons_server#113] [Q&A] [answered]
Hello!

After a certain time of playing for the tribe of Europeans on my computer, artifacts begin to appear on my computer, interfering with the perception of the space of the game.

In this case, the game itself begins to slow down - and then slows down more and more.

How to remove this phenomenon?

Thanks :)

![222](https://user-images.githubusercontent.com/307416/141984690-b99eaeff-6109-4c24-a15c-343999a32d27.png)

Xubuntu 21.10
Widelands 1.0
Europeans Tribe 1.8.11-72

**Comment by Noordfrees** [comment, marked as answer]
Hi @Nick222,
this is a known bug for which has not been investigated yet unfortunately: https://github.com/widelands/widelands/issues/4881
